src/pages/dashboard.js

```diff
@@ -1,13 +1,25 @@
 import React from "react";
 
-import Dashboard from "../components/dashboard";
+import { serverSideTranslations, RequiredNamespaces } from "i18n";
+import Dashboard from "components/dashboard";
 
 function DashboardPage() {
     return <Dashboard />;
 }
 
-DashboardPage.getInitialProps = async () => ({
-    namespacesRequired: ["dashboard", "apps"],
-});
+export async function getStaticProps(context) {
+    // The `locale` prop is undefined in this page for some reason.
+    const { locale, defaultLocale } = context;
+
+    return {
+        props: {
+            ...(await serverSideTranslations(locale ?? defaultLocale, [
```

Comment:
The `locale` prop is undefined in this page for some reason (but not the `defaultLocale`). I think due to the way we render this page for the root path:
https://github.com/cyverse-de/sonora/blob/961672814c603ce020b46293e70e44985b7882ec/src/server/index.js#L174-L176